src/main/groovy/net/wasdev/wlp/gradle/plugins/tasks/AbstractTask.groovy

```diff
@@ -70,4 +101,201 @@ abstract class AbstractTask extends DefaultTask {
+        }
+        else{
+            jvmOptionsFile = new File("/src/main/liberty/config/jvm.options")
+        }
+
+        if(project.liberty.serverEnv != null && project.liberty.serverEnv.length() != 0){
+            serverEnv = new File(project.liberty.serverEnv)
+        }
+        else{
+            serverEnv = new File("/src/main/liberty/config/server.env")
+        }
+    }
+
+    //Gets the loose bootstrap and jvm properties from the build file
+    protected void setProperties(){
+        if(project.liberty.bootstrapProperties != null && !project.liberty.bootstrapProperties.isEmpty()){
+            bootstrapProperties = project.liberty.bootstrapProperties
+        }
+        if(project.liberty.jvmOptions != null && !project.liberty.jvmOptions.isEmpty()){
+            jvmOptions = project.liberty.jvmOptions
+        }
+    }
+
+    private void copyFiles(File srcFile, File destFile){
+        def copyAnt = new AntBuilder()
+        copyAnt.copy(file:srcFile.getCanonicalPath(), tofile:destFile.getCanonicalPath(), overwrite:true)
+    }
+
+    /**
+     * @throws IOException
+     * @throws FileNotFoundException
+     */
+    protected void copyConfigFiles() throws IOException {
+
+        setDirs()
+        setProperties()
+
+        String serverDirectory = project.buildDir.toString() + "/wlp/usr/servers/" + project.liberty.serverName
+
+        String serverXMLPath = null
+        String jvmOptionsPath = null
+        String bootStrapPropertiesPath = null
+        String serverEnvPath = null
+
+        if (configDirectory != null) {
+            if(configDirectory.exists()){
+	            // copy configuration files from configuration directory to server directory if end-user set it
+	            def copyAnt = new AntBuilder()
+	            copyAnt.copy(todir: serverDirectory) {
+	                fileset(dir: project.liberty.configDirectory)
+	            }
+	
+	            File configDirServerXML = new File(configDirectory, "server.xml")
+	            if (configDirServerXML.exists()) {
+	                serverXMLPath = configDirServerXML.getCanonicalPath()
+	            }
+	
+	            File configDirJvmOptionsFile = new File(configDirectory, "jvm.options")
+	            if (configDirJvmOptionsFile.exists()) {
+	                jvmOptionsPath = configDirJvmOptionsFile.getCanonicalPath()
+	            }
+	
+	            File configDirBootstrapFile = new File(configDirectory, "bootstrap.properties")
+	            if (configDirBootstrapFile.exists()) {
+	                bootStrapPropertiesPath = configDirBootstrapFile.getCanonicalPath()
+	            }
+	
+	            File configDirServerEnv = new File(configDirectory, "server.env")
+	            if (configDirServerEnv.exists()) {
+	                serverEnvPath = configDirServerEnv.getCanonicalPath()
+	            }
+            }
+            else{
+                println('WARNING - ConfigDir was configured but not found: ' + configDirectory)
```

Comment:
WARNING: The configDirectory attribute was configured but the directory is not found: